src/main/java/org/generationcp/middleware/pojos/germplasm/ImportedCrossParent.java

```diff
@@ -0,0 +1,47 @@
+package org.generationcp.middleware.pojos.germplasm;
+
+import org.pojomatic.annotations.AutoProperty;
+
+import java.io.Serializable;
+
+@AutoProperty
+public class ImportedCrossParent implements Serializable {
```

Comment:
We have so many DTOs already in Middleware related to Germplasm. Let's reuse StudyGermplasm DTO here:
* designation
* germplasmId
* position = plotNo